python/ray/tune/tests/test_result_grid.py

```diff
@@ -224,6 +224,26 @@ def f(config):
     assert not any(key in representation for key in AUTO_RESULT_KEYS)
 
 
+def test_result_grid_repr(ray_start_2_cpus):
+    def f(config):
```

Comment:
We can actually mock an error for one of the trials, then can test that it shows up in the repr later:

```python
if config["x"] == 1:
    raise RuntimeError
```

Reply:
I see. So the error thrown inside the trainable will be captured and stored in the Result object right?

Reply:
Yep!